cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/rotate-control.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { rotateCurrentFrame } from 'actions/annotation-actions';
+
+import RotateControlComponent from 'components/annotation-page/standard-workspace/controls-side-bar/rotate-control';
+
+import {
+    Rotation,
+} from 'cvat-canvas';
+
+interface DispatchToProps {
+    rotateFrame(angle: Rotation): void;
```

Comment:
Let's add this callback to ``ControlsSideBarContainer`` to avoid extra container.